app/main.py

```
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for person in people:
+        current = Person(person["name"], person["age"])
```

This violates checklist item #4: "When accessing values in dictionaries, prefer dict.get() over explicit key checks to simplify code and avoid KeyError [CHECKLIST ITEM #4]."

You directly index the input dict with `person["name"]` and `person["age"]` when constructing Person instances. Use `person.get('name')` and `person.get('age')` (and validate or fail loudly if these required fields are missing) to follow the checklist and avoid potential KeyError.